src/osemgrep/language_server/InitializeRequest.mli

```diff
@@ -0,0 +1,4 @@
+val on_request :
```

Comment:
Same, please add comments in .mli for the user of this module, to better understand the whole code organization.